Teams section update / cleanup (#76)
srobo/runbook: docs/teams/mentoring.md

  authors: Fiona Gillan
  url: https://github.com/fgillan/sr-team-coord/wiki/Mentoring
---

# Mentoring

> When collecting information on volunteers wanting to be mentors, it needs to collect:

Reviewer: ```suggestion
When collecting information on volunteers wanting to be mentors, we needs to collect:
```

> * Confirm over 18

Reviewer: ```suggestion
* Confirmation that the volunteer is over 18
```

> There are currently five Local Team Coordinators who cover a rough geographic area of the country.  They will be responsible for you as a Mentor and will hopefully be able to arrange social events throughout the year where you can meet other Mentors.
>
> Think this is something you would like to be involved in?  We would ideally allocate a Mentor to each of the 54 teams so we need as many of you to volunteer as possible…so get involved and help guide a team to success in this year’s competition!

> * Once replies obtained, make ‘Mentor Sign-Up’ spreadsheet with this info and match Mentors with their teams.

Reviewer: ```suggestion
Once we have the replies, make ‘Mentor Sign-Up’ spreadsheet with this info and match Mentors with their teams.

The steps are then:

```
Maybe also clarify whether that's replies from volunteers or from teams (or is it both?)
